RELEASE/scripts/autoscend/combat/auto_combat_default_stage3.ash

```diff
@@ -237,8 +237,22 @@ string auto_combatDefaultStage3(int round, monster enemy, string text)
 			return useSkill($skill[Curse Of Weaksauce]);
 		}
 
+		//HP reduction if the monster has high HP
+		if(monster_hp() > 1500)
```

Comment:
this will trigger for the battlefield bosses they both have 2k hp just fyi. not urgent

Reply:
Yes, this was actually chosen deliberately. Right now we have a glut of leaves for those that have the Guide to Burning Leaves because we don't do much with them. I figure cutting those boss' hp in half is an efficient use of the leaves. If there is a reason why we wouldn't necessarily want to do that, I'm open to putting it to 2000 so it ignores the war bosses.